blocks/filter.py

```diff
@@ -88,12 +96,20 @@ class VariableFilter(object):
 
     """
     def __init__(self, roles=None, bricks=None, each_role=False, name=None,
-                 application=None):
+                 name_re=None, applications=None, one=True):
         self.roles = roles
-        self.bricks = bricks
+        if not isinstance(bricks, list):
```

Comment:
@bartvm There is always `collections.abc.Iterable` but I guess the inverse works better.